examples/Example13/example13.cu

```diff
@@ -120,7 +120,13 @@ __global__ void InitPrimaries(ParticleGenerator generator, int startEvent, int n
     track.initialRange = -1.0;
 
     track.pos = {0, 0, 0};
-    track.dir = {1.0, 0, 0};
+    if (rotatingParticleGun) {
+      const double phi = 2. * 3.14159 * i / numEvents;
+      const double eta = -5. + 10. * i / numEvents;
```

Comment:
This introduces a rather weird correlation eta-phi. How about a good old uniform theta/phi gun: 
```
phi = 2*M_PI*rnd();
theta= acos(1.-2.*rnd());
track.dir ={ sin(theta)*cos(phi), sin(theta)*sin(phi), cos(theta) };
```
This is normalized, de-correlated, and reproducible if we start the rng from the same seed.

Reply:
I preferred a weird correlation over shooting everything in positive x, but I'm willing to introduce more randomness. 🙂

Reply:
I wouldn't go uniform in theta, though. Collisions are "uniform" in eta, no in theta. We would be skewing the probing of the geometry towards the centre, where we don't hit that often in the real world.

Reply:
Sure, generating uniform eta is better